App/Components/Styles/ModalStyle.js

```diff
@@ -0,0 +1,52 @@
+import { StyleSheet } from 'react-native'
+import { ApplicationStyles, Colors } from '../../Themes/'
+
+export default StyleSheet.create({
+  ...ApplicationStyles.screen,
+  container: {
+    flex: 1,
+    marginTop: 40
```

Comment:
Sorry, I just copied from another component's styles. I'm not yet familiar with your style library.